e2e/run.bash

```diff
@@ -6,8 +6,17 @@ export HACK_DIR
 
 echo ">>> Waiting for capsule-proxy pod to be ready for accepting requests"
 kubectl --namespace capsule-system wait --for=condition=ready --timeout=320s pod -l app.kubernetes.io/instance=capsule-proxy
+
 echo ">>> Waiting for capsule pod to be ready for accepting requests"
 kubectl --namespace capsule-system wait --for=condition=ready --timeout=320s pod -l app.kubernetes.io/instance=capsule
 
-echo ">>> Starting test suite"
+echo ">>> Waiting for metrics-server pod to be ready for listing metrics"
+kubectl --namespace metrics-system wait --for=condition=ready --timeout=320s pod -l app.kubernetes.io/instance=metrics-server
+until [[ $(kubectl get --raw "/apis/metrics.k8s.io/v1beta1/nodes" 2>/dev/null | jq type 2>/dev/null) == "\"object\"" ]]
```

Comment:
This is nice, so we should be able to avoid the flakiness of not-yet-ready metrics. However, we got [a failed test](https://github.com/clastix/capsule-proxy/pull/141/checks?check_run_id=3621599975).

Reply:
Line nr. 20 solution is good, we just need to remember if we add or remove KinD nodes, the expected value must be updated too.

Reply:
I will compare with the number of nodes, in that case we do not need to update it anymore